website/pages/docs/support.md

```diff
@@ -0,0 +1,16 @@
+# Support
+
+Several levels and channels of support are available.
+
+## Community
+
+Community support is free and available to everyone.
+
+[Join our Discord](https://discord.gg/8qZ7Y4Z) to ask questions and get help from both us and the community.
+
+[GitHub Issues](https://github.com/cloudquery/cloudquery/issues) for bug reports, feature requests, and/or questions.
+
+## Enterprise
+
+If you need an SLA, private communication channel, and/or prioritization of resources, plugins, or issues. Please reach out to us at 'support' at our domain to discuss an enterprise support agreement.
```

Comment:
```suggestion
If you need an SLA, a private communication channel, and prioritization of resources, plugins, or issues, please reach out to us at `support` at our domain to discuss an enterprise support agreement.
```

Also, maybe write `cloudquery.io` instead of `domain`?